### PR TITLE
Arepo docs fix

### PR DESCRIPTION
**Pull request by AlexHls**
### :pencil: Description

**Type:** :beetle: `bugfix` 

Fix to failing docs build.

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

**Review by wkerzendorf — APPROVED** [review]

**Comment by epassaro** [comment]
Read the instructions at the pull request body @AlexHls

**Comment by epassaro** [comment]
You are commiting with the `alex.holas@gmx.de` email @AlexHls , please add that to `.mailmap`

**Comment by epassaro** [comment]
The label required to build the docs is this, it's not necessary to write anything in the commit name.
  
image

File: .mailmap [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Adam Suban-Loewen <adam.subanloewen@gmail.com>
 
 Alexander Holas <alexander.holas@h-its.org>
+Alexander Holas <alex.holas@gmx.de>
```
Comment (epassaro) [inline .mailmap:4]: I think should be:

`Alexander Holas <alexander.holas@h-its.org> Alexander Holas <alex.holas@gmx.de>`

**Pull request merged**